plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/hand-picked-products-control.tsx

```diff
@@ -148,6 +148,34 @@ const HandPickedProductsControl = ( {
 		return decodeEntities( product?.name ) || '';
 	};
 
+	return (
+		<FormTokenField
+			displayTransform={ transformTokenIntoProductName }
+			label={ __( 'Select Products', 'woocommerce' ) }
```

Comment:
When the control is displayed as a part of filters, the label should be `Hand-picked products`, and when it's used as the Hand-picked collection-specific control, it should be `Select products`. What would be the best way to contextualize this label? Should we just pass an extra prop?

cc: @imanish003 @kmanijak @xristos3490

Reply:
I am not sure if we should change it based on this. Maybe we can keep just one of these? For example, Maybe we can use "Hand-picked Products" for both cases? But it's worth confirming with Jill 🙂

If we decide to change it, then I think it's okay to pass an extra prop for now. For example if `label` prop is passed then we use it, otherwise we use the default one. Because `HandPickedProductsControl` is just a reusable React component from my POV.

Reply:
> I am not sure if we should change it based on this. Maybe we can keep just one of these? For example, Maybe we can use "Hand-picked Products" for both cases? But it's worth confirming with Jill 

Personally, I'd opt for leaving the `Select Products` if we decide to go with one title (see the screenshots below). No strong opinion here, though. @j111q what do you think? Should we keep one title for both scenarios or go with context-specific ones?

| Hand-Picked collection control | Filters drop-down | Filters form |
| ------------- | ------------- | ------------- |
| <img width="283" alt="Screenshot 2024-09-09 at 16 12 19" src="https://github.com/user-attachments/assets/ee6eff4a-e520-4703-a787-a69797a8be99"> | <img width="282" alt="Screenshot 2024-09-09 at 16 12 51" src="https://github.com/user-attachments/assets/92598f02-e6a4-4d4a-adaa-77de16e3e10b"> | <img width="284" alt="Screenshot 2024-09-09 at 16 12 59" src="https://github.com/user-attachments/assets/b576c782-7311-41b7-896b-f64d8ad509f8"> |

Reply:
I'm totally for the unified label. There's no reason to contextualize which just makes the control more complex (code-wise and UX-wise). `Select products` sounds good to me!

Reply:
Hi, it makes sense to me too to use the same label in both contexts 👍 That being said, could we change the copy to `Select specific products` so it's a little clearer?

Reply:
Thanks, @j111q! Looking at the other filters now, I wonder if we should adjust the phrasing for a consistent tone, as `Select specific products` would be the only one that calls to action. What if we go with `Selected Products` or `Products Selection` so that it matches other filters?

Reply:
Oh that's a great point. Maybe we could go with `Hand-picked Products` then for super consistency? 😆

Reply:
Following a quick discussion with Jill on Slack, we’ve decided to use `Hand-picked Products` across both contexts. Additionally, the collection description will be updated to `Select specific products to recommend to customers` to clarify the purpose. 🙌

Reply:
Updated in 7f4e0e8bc80b65d9aa1767f083700a69ca5cad23.